server/src/middleware/jwt.middleware.ts

```diff
@@ -16,7 +17,7 @@ const jwtMiddleware = (req: Request, res: Response, next: NextFunction) => {
     const userData = tokenService.validateAccessToken(accessToken);
     if (!userData) return next(ApiError.unauthorized("The token is not valid"));
 
-    // req.user = userData;
+    req.user = new UserDto(userData);
```

Comment:
to do this, I extended the Request type in express/index.d.ts

declare global{
    namespace Express {
        export interface Request {
            user?: UserDto;
        }
    }
}